### ISSUE TITLE
Minify styles removes @keyframes

### ISSUE THREAD

**Issue by jamieburchell** [jakearchibald/svgomg#388]
This might be an issue with the underlying lib, but I've found that with the minify styles option turned on the style block changes from this:

```
<style>.VAWiBvXU_0{stroke-dasharray:326 328;stroke-dashoffset:327;animation:VAWiBvXU_draw 13333ms linear 0ms forwards;}@keyframes VAWiBvXU_draw{100%{stroke-dashoffset:0;}}</style>
```

to this:

```
<style>.VAWiBvXU_0{stroke-dasharray:326 328;stroke-dashoffset:327;animation:VAWiBvXU_draw 13333ms linear 0ms forwards;}</style>
```

**Comment by XhmikosR** [comment]
Yeah, this is an issue with svgo. Not sure if it's fixed in v2.8.0, we updated to it recently. Can you try with this? https://svgomg.netlify.app/

If it's fixed in svgo v3.0.0, we'll need someone to make a proper patch to update to that version here too.

**Comment by XhmikosR** [comment]
Please make an upstream issue https://github.com/svg/svgo/issues with the source SVG file. It seems there are still some known keyframes issues https://github.com/svg/svgo/issues?q=is%3Aissue+keyframes
